src/frontend/packages/lib_components/src/hooks/stores/useJwt/index.ts

```diff
@@ -1,14 +1,17 @@
 import { create } from 'zustand';
-import { persist } from 'zustand/middleware';
 
 import { DecodedJwt } from '../../../types/jwt';
 import { decodeJwt } from '../../../utils/decodeJwt';
 
+const JWT_KEY = 'JWT';
+const REFRESH_JWT_KEY = 'REFRESH_JWT';
 interface JwtStoreInterface {
   jwt?: string;
   refreshJwt?: string;
   internalDecodedJwt?: DecodedJwt;
+  getJwt: () => string | undefined;
```

Comment:
As @lunika mention, prefer `get jwt()`.
IMO, we should add some testcases on this store.